src/main/java/com/unosquare/cvgenerator/controller/HelloController.java

```diff
@@ -1,29 +1,53 @@
 package com.unosquare.cvgenerator.controller;
 
 import com.unosquare.cvgenerator.model.dto.GreetingDTO;
+import com.unosquare.cvgenerator.model.dto.TemplateDetailDTO;
 import com.unosquare.cvgenerator.model.view.GreetingView;
 import com.unosquare.cvgenerator.service.GreetingService;
+import com.unosquare.cvgenerator.service.TemplateService;
 import com.unosquare.cvgenerator.util.MapperUtil;
 import lombok.RequiredArgsConstructor;
-import org.springframework.stereotype.Controller;
-import org.springframework.ui.Model;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
 
-@Controller
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Slf4j
+@RestController
 @RequiredArgsConstructor
 public class HelloController {
 
     private final GreetingService greetingService;
+    private final TemplateService templateService;
     private final MapperUtil mapperUtil;
 
     @GetMapping("/hello/{id}")
-    public String sayHello(@PathVariable("id") Integer id, Model theModel) {
+    public String sayHello(@PathVariable("id") Integer id) {
```

Comment:
I get the impression this is just for testing and some of the code should be moved to the TemplateController; but yeah I agree it might be worth cleaning up.